features/bevragen/persoon/gezag/gba/gezag-meerderjarige-gba.feature

```diff
@@ -0,0 +1,721 @@
+      | naam                             | waarde           |
+      | type                             | GezamenlijkGezag |
+      | minderjarige.burgerservicenummer | 000000012        |
+      | ouder.burgerservicenummer        | 000000024        |
+      | derde.burgerservicenummer        | 000000048        |
+
+    Scenario: persoon heeft van rechtswege gezamenlijk gezag over enkele van de minderjarige kinderen van diens partner
+      Gegeven de persoon met burgerservicenummer '000000048' heeft een 'kind' met de volgende gegevens
+      | burgerservicenummer (01.20) |
+      | 000000012                   |
+      En de persoon heeft een 'kind' met de volgende gegevens
+      | burgerservicenummer (01.20) |
+      | 000000024                   |
+      En de persoon heeft een 'kind' met de volgende gegevens
+      | burgerservicenummer (01.20) |
+      | 000000036                   |
+      En de persoon heeft een 'partner' met de volgende gegevens
+      | burgerservicenummer (01.20) |
+      | 000000061                   |
+      En voor de persoon met burgerservicenummer '000000048' gelden de volgende gezagsrelaties
+      | bsnMinderjarige | soortGezag | bsnMeerderjarige |
+      | 000000012       | GG         | 000000024        |
+      | 000000024       | GG         | 000000024        |
+      | 000000036       | GG         | 000000024        |
+      En de persoon met burgerservicenummer '000000012' heeft de volgende gegevens
+      | naam                  | waarde   |
+      | geboortedatum (03.10) | 20180219 |
+      En voor de persoon met burgerservicenummer '000000012' gelden de volgende gezagsrelaties
+      | bsnMinderjarige | soortGezag | bsnMeerderjarige |
+      | 000000012       | GG         | 000000048        |
+      | 000000012       | GG         | 000000061        |
+      En de persoon met burgerservicenummer '000000024' heeft de volgende gegevens
+      | naam                  | waarde   |
+      | geboortedatum (03.10) | 20191014 |
+      En voor de persoon met burgerservicenummer '000000024' gelden de volgende gezagsrelaties
+      | bsnMinderjarige | soortGezag | bsnMeerderjarige |
+      | 000000036       | GG         | 000000061        |
+      | 000000036       | GG         | 000000048        |
+      En de persoon met burgerservicenummer '000000036' heeft de volgende gegevens
+      | naam                  | waarde   |
+      | geboortedatum (03.10) | 20210516 |
+      En voor de persoon met burgerservicenummer '000000036' gelden de volgende gezagsrelaties
+      | bsnMinderjarige | soortGezag | bsnMeerderjarige |
+      | 000000036       | GG         | 000000048        |
+      | 000000036       | GG         | 000000073        |
+      En de persoon met burgerservicenummer '000000061' heeft een 'partner' met de volgende gegevens
+      | burgerservicenummer (01.20) |
+      | 000000048                   |
+      En de persoon met burgerservicenummer '000000061' heeft geen gezagsrelaties
```

Comment:
beetje verwarrend misschien? Bedoel je dit in de zin van: geen gezagsrelaties als de gezagsmodule met 61 wordt bevraagd?

Reply:
dat is inderdaad wat er bedoeld wordt. de gezagsmodule levert geen gezagsrelaties.
Hetzelfde geldt voor de stap "En voor de persoon met burgerservicenummer '000000024' gelden de volgende gezagsrelaties". Dit betekent de gezagsmodule levert deze gezagsrelaties.

Zou je die stappen liever wijzigen zodat expliciet is dat we hier bedoelen wat de gezagsmodule levert?

Reply:
Ik twijfel. Ik vraag me af of dat bijdraagt aan de begrijpelijkheid van het scenario/ de functionaliteit en vereist begrip van de werking van de gezagsmodule. Misschien is deze tight coupling nodig. Morgen even over hebben?